src/script/lua_api/l_object.cpp

```diff
@@ -1724,7 +1724,11 @@ int ObjectRef::l_set_sky(lua_State *L)
 	SkyboxParams sky_params = player->getSkyParams();
 	bool is_colorspec = is_color_table(L, 2);
 
-	if (lua_istable(L, 2) && !is_colorspec) {
+	// reset if empty
+	if (lua_isnoneornil(L, 2) && lua_isnone(L, 3))
+		sky_params = SkyboxDefaults::getSkyDefaults();
```

Comment:
`if` braces must be consistent, either `if (...) { ... } else { ... }` or `if (...) ... else ...` but not `if (...) ... else { ... }`